SUMMARY.md

```diff
@@ -12,28 +12,30 @@
   * [Ingress Examples](ingress-examples.md)
   * [Cloud Billing Integrations](cloud-integration.md)
     * [Multi-Cloud Integrations](multi-cloud.md)
-    * [AWS Cloud Integration](aws-cloud-integrations.md)
+    * [AWS Cloud Billing Integration](aws-cloud-integrations.md)
+      * [AWS Cross-Account Billing Integration](aws-cur-setup.md)
       * [AWS Out of Cluster](aws-out-of-cluster.md)
       * [AWS Spot Instances](aws-spot-instances.md)
       * [AWS Node Price Reconciliation Methodology](aws-node-price-reconcilitation-methodology.md)
-    * [Azure Cloud Integration](azure-out-of-cluster.md)
+    * [Azure Cloud Billing Integration](azure-out-of-cluster.md)
       * [Azure Rate Card Configuration](azure-config.md)
-    * [GCP Cloud Integration](gcp-out-of-cluster.md)
+    * [GCP Cloud Billing Integration](gcp-out-of-cluster.md)
       * [Creating a Google Service Account](google-service-account-thanos.md)
   * [Multi-Cluster](multi-cluster.md)
-    * [Federated ETL](federated-etl.md)
+    * [ETL Federation (preferred)](federated-etl.md)
       * [Backups and Alerting](federated-etl-backups-alerting.md)
-    * [Thanos Federation](thanos-setup.md)
+    * [Thanos Federation (fully supported)](thanos-setup.md)
```

Comment:
Why the "fully supported" qualification here? Seems a bit odd for a document/page title in a product's official documentation.